app/components/challenge-workspace.js

```diff
@@ -186,7 +187,10 @@ export default Component.extend({
     },
 
     async ejecutar(pasoAPaso = false) {
-      await this.pilasBlockly.send('ejecutar', pasoAPaso);
+      this.set('areExpectationsRunning', true)
+      await sleep(50) // This enables the component to rerender
+      await this.pilasBlockly.actions.ejecutar.call(this.pilasBlockly, pasoAPaso)
```

Comment:
Esta es la forma medio hackosa que encontré para poder _awaitear_ una action. Se necesita hacer `call` para establecer bien el valor de `this` al correrlo de esta manera.

Leer más abajo.

Reply:
Me recontracago en Ember. GRAN investigación, Eze